File: NzolaSPA/src/app/dtos/denuncia/denuncia.dto.ts
export class CriarDenunciaDto {
  tipoEntidade: number = 0;
  idEntidade: number = 0;
  motivo: string = '';
  descricao: string = '';
  denuncianteId: number = 0;
}

export class DenunciaDto {
  id: number = 0;
  tipoEntidade: number = 0;
  idEntidade: number = 0;
  motivo: string = '';
  descricao: string = '';
  denuncianteId: number = 0;
  nomeDenunciante: string = '';
  dataDenuncia: string = '';
  estadoDenuncia: number = 0;
}

File: NzolaWebAPI/Dtos/Denuncia/DenunciaDto.cs
using NzolaWebAPI.Models.Enums;

namespace NzolaWebAPI.DTOs.Denuncia
{
    public class DenunciaDto
    {
        public int Id { get; set; }
        public TipoEntidade TipoEntidade { get; set; }
        public int IdEntidade { get; set; }
        public string Motivo { get; set; } = string.Empty;
        public string? Descricao { get; set; }
        public int DenuncianteId { get; set; }
        public string? NomeDenunciante { get; set; }
        public DateTime DataDenuncia { get; set; }
        public EstadoDenuncia EstadoDenuncia { get; set; }
    }
}
